databases/week1/homework.sql

```diff
@@ -0,0 +1,95 @@
+-- Task 1 Find out how many tasks are in the task table
+SELECT 
+  COUNT(*) 
+FROM 
+  task;
+
+-- Task 2 Find out how many tasks in the task table do not have a valid due date
+SELECT 
+  COUNT(*) 
```

Comment:
COUNT(Id) as task_amount
Pointer
Add a column name to all columns created by you

Reply:
Fixed :) Thank you for the feedback!